airflow/dag_processing/manager.py

```diff
@@ -477,7 +491,8 @@ def _refresh_dag_bundles(self):
 
             new_file_paths = [f for f in self._file_paths if f.bundle_name != bundle.name]
             new_file_paths.extend(
-                DagFileInfo(path=path, bundle_name=bundle.name) for path in bundle_file_paths
+                DagFileInfo(path=path, bundle_name=bundle.name, bundle_version=bundle_model.version)
```

Comment:
Same here. We want `_file_stat` to be "versionless", which is easier if we just leave these all as None.